### ISSUE TITLE
Doesn't support . repetition

### ISSUE THREAD

**Issue by exchgr** [jcartledge/vintage-sublime-surround#2]
When I press the period key in vim, tpope's surround plugin's action repeats itself if it was the last action. The period key also repeats the last action in Sublime Text's Vintage Mode, but it doesn't work with this plugin.

**Comment by jcartledge** [comment]
You're right - this is very annoying. I'm not sure when I'll get time to look at it but I'd love to get it fixed.

**Comment by jcartledge** [comment]
OK, some progress: this mostly works in the ST3 branch of sublime-surround (see https://github.com/jcartledge/sublime-surround/pull/10) although it's broken when repeating `ys<motion>`. I'll try to get that working before I release the update, but if you want to test it please go ahead.

Note that the ST3 branch works fine in ST2 as well.

**Comment by jcartledge** [comment]
I've just merged the ST3 branch so it should update through package control. Repeating `ys<motion>` is still broken, but other repeats work well. Note that surround.vim only supports this through repeat.vim (another @tpope plugin which enables "repeating supported plugin maps with '.'") Not sure how viable this approach is in Vintage.
